webapp/app/elster_client/elster_client.py

```diff
@@ -42,14 +42,22 @@
 
 def send_to_erica(*args, **kwargs):
     logger.info(f'Making Erica POST request with args {args!r}')
-    if Config.USE_MOCK_API:
-        from tests.elster_client.mock_erica import MockErica
-        response = MockErica.mocked_elster_requests(*args, **kwargs)
-    else:
-        headers = {'Content-type': 'application/json'}
-        response = requests.post(*args, headers=headers, timeout=_REQUEST_TIMEOUT, **kwargs)
-    logger.info(f'Completed Erica POST request with args {args!r}, got code {response.status_code}')
-    return response
+    try:
+        if Config.USE_MOCK_API:
+            from tests.elster_client.mock_erica import MockErica
+            response = MockErica.mocked_elster_requests(*args, **kwargs)
+        else:
+            headers = {'Content-type': 'application/json'}
+            response = requests.post(*args, headers=headers, timeout=_REQUEST_TIMEOUT, **kwargs)
+            logger.info(f'Completed Erica POST request with args {args!r}, got code {response.status_code}')
+        return response
+    except RequestException as error:
+        logger.info(f'Completed Erica POST request with args {args!r}, got code {error.response.status_code}')
+        if error is TimeoutError:
+            raise EricaRequestTimeoutError(error)
+        if error is ConnectionError:
+            raise EricaRequestConnectionError(error)
+        raise GeneralEricaError()
```

Comment:
Could we add tests for these errors? We could mock the requests.post or mocked_elster_requests function to raise an TimeoutError, ConnectionError or a RequestException and then check that the corresponding Elster error is raised